### PR TITLE
Add klyva to results.

### PR DESCRIPTION
**Pull request by merisbahti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -382,6 +382,19 @@ in your browser, and click the button very quickly. (check the console log)
     check with intensive auto increment
       ✓ check 8: updated properly with auto increment (3000 ms)
       ✓ check 9: no tearing with auto increment
+  klyva
+    check with events from outside
+      ✓ check 1: updated properly (7993 ms)
+      ✓ check 2: no tearing during update
+      ✕ check 3: ability to interrupt render (1 ms)
```
Comment (dai-shi) [inline README.md:389]: What are the console log for the real delays values?
This fails if the avg delay is not less then 300ms.

**Review by dai-shi — COMMENTED** [review]
Hi Meris, glad you got interested in trying.

File: README.md (hunk 1 above)
Comment (merisbahti) [reply README.md:389]: oh I didn't see that it failed, I was sure that it didnt. hmm, what do you mean?

Comment (merisbahti) [reply README.md:389]: what should I do to trigger the logs?

Comment (merisbahti) [reply README.md:389]: Trying to run it again yields this:
```
[I] meris@pop-os ~/c/will-this-react-global-state-work-in-concurrent-mode (master) [1]> PORT=8080 yarn run jest
yarn run v1.22.5
$ cross-env PORT=8080 BABEL_ENV=jest jest
  console.log
    klyva [ 190, 184, 184, 185, 183, 187 ]

      at Object.<anonymous> (__tests__/all_spec.js:73:17)

 FAIL  __tests__/all_spec.js (24.393 s)
  klyva
    check with events from outside
      ✓ check 1: updated properly (3130 ms)
      ✓ check 2: no tearing during update (2 ms)
      ✓ check 3: ability to interrupt render
      ✓ check 4: proper update after interrupt (1151 ms)
    check with useTransition
      ✓ check 5: updated properly with transition (5184 ms)
      ✓ check 6: no tearing with transition (1 ms)
      ✕ check 7: proper branching with transition (5964 ms)
    check with intensive auto increment
      ✓ check 8: updated properly with auto increment (3092 ms)
      ✕ check 9: no tearing with auto increment (4 ms)

  ● klyva › check with useTransition › check 7: proper branching with transition

    TimeoutError: Element .count:nth-of-type(1) (text: "6") not found

      waiting for function failed: timeout 5000ms exceeded

  ● klyva › check with intensive auto increment › check 9: no tearing with auto increment
 
```

Comment (dai-shi) [reply README.md]: > klyva [ 190, 184, 184, 185, 183, 187 ]

It's totally fine!

**Review by dai-shi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (merisbahti) [reply README.md]: i might have pasted an old log, i did the measurements and my computer fell asleep, whcih caused test 3 to fail i think

**Pull request merged**